cdc/model/changefeed.go

```diff
@@ -455,6 +462,18 @@ func (info *ChangeFeedInfo) fixMySQLSinkProtocol() {
 	}
 }
 
+func (info *ChangeFeedInfo) fixEnableOldValue() {
+	uri, err := url.Parse(info.SinkURI)
+	if err != nil {
+		// this is impossible to happen, since the changefeed registered successfully.
+		log.Warn("parse sink URI failed", zap.Error(err))
```

Comment:
Maybe more info is needed here. For example, the changefeed and sink-uri(Desensitized).

Reply:
`util.MaskSinkURI(info.SinkURI),` can be called here to desensitize the url. 
print url here may cause leak.